src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -704,14 +723,16 @@ private EndpointBitrateAllocation(
          */
         void allocate(long maxBps, int maxQuality)
         {
-            if (!forwarded || rates.length == 0)
+            if (rates.length == 0)
             {
                 return;
             }
 
-            maxQuality = selected ? Math.min(maxQuality, rates.length - 1) : 0;
+            maxQuality = Math.min(maxQuality, optimalIdx);
```

Comment:
does `maxQuality` refer to an index?  i think it'd be good to rename it to `maxQualityIdx`